include/odp/api/spec/crypto.h

```diff
@@ -585,15 +585,15 @@ int odp_crypto_auth_capability(odp_auth_alg_t auth,
  * default values.
  *
  * @param param             Session parameters
- * @param session           Created session else ODP_CRYPTO_SESSION_INVALID
  * @param status            Failure code if unsuccessful
  *
- * @retval 0 on success
- * @retval <0 on failure
+ * @retval created session on success
+ * @retval ODP_CRYPTO_SESSION_INVALID on failure
  */
-int odp_crypto_session_create(odp_crypto_session_param_t *param,
-			      odp_crypto_session_t *session,
-			      odp_crypto_ses_create_err_t *status);
+odp_crypto_session_t odp_crypto_session_create(odp_crypto_session_param_t
+					       *param,
+					       odp_crypto_ses_create_err_t
```

Comment:
@Bill-Fischofer-Linaro because we do not use `odp_errno` anywhere else. We can change that later.